### PR TITLE
Added .gitkeep file to empty directory.

### PR DESCRIPTION
**Pull request by rsundar**
First milestone of the Object Oriented Programming - making a Tic Tac Toe game in ruby.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "Hello World!"
```
Comment (bolah2009) [inline bin/main.rb:3]: Let's make the `main.rb` file executable.

Reminder from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)

> Make your main.rb file displaying “Hello world!” when you use bin/main.rb as a bash command.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Please make the file executable.

> bin - with the [executable file] https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) called main.rb.

Make these changes and submit another CRR

File: bin/main.rb (hunk 1 above)
Comment (rsundar) [reply bin/main.rb:3]: Thank you, will make the appropriate changes and resubmit for code review.

**Review by AdnanAfsari — APPROVED** [review]
**Status: Approved**

Looks good, its a 👍 .

Happy Coding
Adnan Afsari

**Comment by rsundar** [comment]
> **Status: Approved**
> 
> Looks good, its a  .
> 
> Happy Coding
> Adnan Afsari

Thanks @AdnanAfsari @bolah2009 :+1:

**Pull request merged**